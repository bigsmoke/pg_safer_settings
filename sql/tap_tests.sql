begin;

create extension pg_safer_settings cascade;

call test__pg_db_setting();
call test__pg_safer_settings_table();

rollback;
