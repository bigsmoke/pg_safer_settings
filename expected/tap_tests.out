begin;
create extension pg_safer_settings cascade;
NOTICE:  installing required extension "pg_utility_trigger_functions"
NOTICE:  installing required extension "hstore"
call test__pg_db_setting();
call test__pg_safer_settings_table();
rollback;
